src/InitApolloClient.re

```diff
@@ -0,0 +1,20 @@
+let createClient =
+    (
+      ~cache,
+      ~link,
+      ~ssrMode=false,
+      ~ssrForceFetchDelay=0,
+      ~connectToDevTools=Js.Nullable.undefined,
+      ~queryDeduplication=true,
```

Comment:
Quick question: are you sure you want to provide the defaults here? I would suggest to let the JS implementation do that, in case something changes there. Which means we should pass a nullable/undefined value.

Reply:
True

Reply:
Yes, good catch